utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/SwaggerTyped.scala

```diff
@@ -54,9 +62,11 @@ sealed trait SwaggerUnknownFallback extends SwaggerTyped
 
 case object SwaggerRecursiveSchema extends SwaggerUnknownFallback
 
-case object SwaggerEnumOfVariousTypes extends SwaggerUnknownFallback
-
 object SwaggerTyped {
+  private val bestEffortJsonEncoder =  BestEffortJsonEncoder(failOnUnkown = true, getClass.getClassLoader)
+  private implicit val decoder: Decoder[List[Any]] = Decoder[Json].map(_.asArray.map(_.toList.map(jsonToAny)).getOrElse(List.empty))
+  private implicit val encoder: Encoder[List[Any]] = Encoder.instance[List[Any]](bestEffortJsonEncoder.encode)
+  private lazy val om = new ObjectMapper()
```

Comment:
let's wrap it in some private class (the enum part) so that no one plays with those encoders here without a good reason ;)